tools/wptrunner/wptrunner/browsers/safari.py

```diff
@@ -193,15 +193,21 @@ def stop(self, force=False):
         if self.kill_safari:
             self.logger.debug("Going to stop Safari")
             for proc in psutil.process_iter(attrs=["exe"]):
-                if (proc.info["exe"] is not None and
-                    os.path.samefile(proc.info["exe"], self.safari_path)):
-                    self.logger.debug("Stopping Safari %s" % proc.pid)
+                try:
+                    if proc.info["exe"] is None or not os.path.samefile(
```

Comment:
Was this reformatted with black by any chance? I honestly find this significantly more difficult to read vs the original (and obviously more difficult to work out what changed since there's a gratuitous formatting change.

Reply:
I'm not sure the diff is much clearer undoing this, but 🤷🏻‍♀️